core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java

```diff
@@ -309,6 +317,17 @@ protected void stopServing() throws Exception {
         LOG.warn("RPC Server shutdown timed out.");
       }
     }
+    if (mRPCExecutor != null) {
+      mRPCExecutor.shutdown();
+      try {
+        mRPCExecutor.awaitTermination(
```

Comment:
Could you please run FileSystemMasterIntegrationTest and validate that it runs about 1 minute long?
If awaiting has adverse effect on the integration test run times, we can maybe consider not making awaitTermination call.

Reply:
no adverse effect observed